extensions/mssql/src/dacfx/dacFxService.ts

```diff
@@ -55,7 +55,18 @@ export class DacFxService implements mssql.IDacFxService {
 	}
 
 	public extractDacpac(databaseName: string, packageFilePath: string, applicationName: string, applicationVersion: string, ownerUri: string, taskExecutionMode: azdata.TaskExecutionMode): Thenable<mssql.DacFxResult> {
-		const params: contracts.ExtractParams = { databaseName: databaseName, packageFilePath: packageFilePath, applicationName: applicationName, applicationVersion: applicationVersion, ownerUri: ownerUri, taskExecutionMode: taskExecutionMode };
+		const params: contracts.ExtractParams = { databaseName: databaseName, packageFilePath: packageFilePath, applicationName: applicationName, applicationVersion: applicationVersion, ownerUri: ownerUri, extractTarget: azdata.ExtractTarget.dacPac, taskExecutionMode: taskExecutionMode };
+		return this.client.sendRequest(contracts.ExtractRequest.type, params).then(
+			undefined,
+			e => {
+				this.client.logFailedRequest(contracts.ExtractRequest.type, e);
+				return Promise.resolve(undefined);
+			}
+		);
+	}
+
+	public importDatabaseProject(databaseName: string, targetFilePath: string, applicationName: string, applicationVersion: string, ownerUri: string, extractTarget: azdata.ExtractTarget, taskExecutionMode: azdata.TaskExecutionMode): Thenable<mssql.DacFxResult> {
+		const params: contracts.ExtractParams = { databaseName: databaseName, packageFilePath: targetFilePath, applicationName: applicationName, applicationVersion: applicationVersion, ownerUri: ownerUri, extractTarget: extractTarget, taskExecutionMode: taskExecutionMode };
 		return this.client.sendRequest(contracts.ExtractRequest.type, params).then(
```

Comment:
you're sending extract request for import. is that correct?

Reply:
yah the terms here are confusing 
in SSDT database --> project is called import. in dacfx apis, its called extract